exercises/practice/parallel-letter-frequency/parallel_letter_frequency_test.go

```diff
@@ -12,32 +12,38 @@ import (
+fiber and liquids—and I might even be said to possess a mind. I am invisible, understand,
+simply because people refuse to see me. Like the bodiless heads you see sometimes in circus sideshows,
+it is as though I have been surrounded by mirrors of hard, distorting glass.
+When they approach me they see only my surroundings, themselves, or figments of their imagination—indeed,
+everything and anything except me.`
 
-	dutch = `Wilhelmus van Nassouwe
-ben ik, van Duitsen bloed,
-den vaderland getrouwe
-blijf ik tot in den dood.
-Een Prinse van Oranje
-ben ik, vrij, onverveerd,
-den Koning van Hispanje
-heb ik altijd geëerd.`
+	dostoevsky = `I am a sick man. ... I am a spiteful man. I am an unattractive man. I believe my liver is diseased.
+However, I know nothing at all about my disease, and do not know for certain what ails me.
+I don't consult a doctor for it, and never have, though I have a respect for medicine and doctors.
+Besides, I am extremely superstitious, sufficiently so to respect medicine, anyway
+(I am well-educated enough not to be superstitious, but I am superstitious). No, I refuse to consult a doctor from spite.
+That you probably will not understand. Well, I understand it, though. Of course,
+I can't explain who it is precisely that I am mortifying in this case by my spite:
+I am perfectly well aware that I cannot "pay out" the doctors by not consulting them;
+I know better than anyone that by all this I am only injuring myself and no one else. But still, if I don't consult a doctor it is from spite. My liver is bad, well then let it hurt even worse!
+`
 
-	us = `O say can you see by the dawn's early light,
-What so proudly we hailed at the twilight's last gleaming,
-Whose broad stripes and bright stars through the perilous fight,
-O'er the ramparts we watched, were so gallantly streaming?
-And the rockets' red glare, the bombs bursting in air,
-Gave proof through the night that our flag was still there;
-O say does that star-spangled banner yet wave,
-O'er the land of the free and the home of the brave?`
+	mark_twain = `You don't know about me without you have read a book by the name of The Adventures of Tom Sawyer;
+but that ain't no matter. That book was made by Mr. Mark Twain, and he told the truth, mainly.
+There was things which he stretched, but mainly he told the truth. That is nothing. I never seen anybody but lied one time or another,
+without it was Aunt Polly, or the widow, or maybe Mary. Aunt Polly - Tom's Aunt Polly, she is - and Mary,
+and the Widow Douglas is all told about in that book, which is mostly a true book, with some stretchers, as I said before.`
+
+	gary_provost = `This sentence has five words. Here are five more words. Five-word sentences are fine.
+But several together become monotonous. Listen to what is happening. The writing is getting boring.
+The sound of it drones. It’s like a stuck record. The ear demands some variety. Now listen. I vary the sentence length,
```

Comment:
```suggestion
The sound of it drones. It's like a stuck record. The ear demands some variety. Now listen. I vary the sentence length,
```

Since you are using `'` for prepositions in the other texts, let's use it here too instead of `’`. `'` is more common in source code. They look very similar, but both vim and vscode were giving me warnings about this.